resolwe_bio/processes/variant_calling/gatk_variant_filtration.py

```diff
@@ -83,6 +83,29 @@ class Input:
             "specify filter names 'FS' and 'DP'.",
             required=False,
         )
+        genotype_filter_expressions = ListField(
+            StringField(),
+            label="Expressions used with FORMAT field to filter",
+            description="Similar to the INFO field based expressions, but used on the FORMAT "
+            "(genotype) fields instead. VariantFiltration will add the sample-level FT tag to "
+            "the FORMAT field of filtered samples (this does not affect the record's FILTER tag). "
+            "One can filter normally based on most fields (e.g. 'GQ < 5.0'), but the GT "
+            "(genotype) field is an exception. We have put in convenience methods so that "
+            "one can now filter out hets ('isHet == 1'), refs ('isHomRef == 1'), or homs "
+            "('isHomVar == 1'). Also available are expressions isCalled, isNoCall, isMixed, "
+            "and isAvailable, in accordance with the methods of the Genotype object."
+            "To filter by alternative allele depth, use the expression: "
+            "'vc.getGenotype('sample_name').getAD().1 < n",
```

Comment:
Perhaps add what `sample_name` and `n` stand for `vc.getGenotype('sample_name').getAD().1 < n`. 

Also the example should end with `'` and the description with a period.